src/status_im/utils/money.cljs

```diff
@@ -90,6 +90,20 @@
   (when-let [bn (bignumber n)]
     (.dividedBy bn (bignumber (from-decimal decimals)))))
 
+(defn unit->token [n decimals]
+  (when-let [bn (bignumber n)]
+    (.times bn (bignumber (from-decimal decimals)))))
+
+(defn formatted->internal [n symbol decimals]
```

Comment:
Can you add some doc? Also why does `internal` refers to?

Reply:
Yes, indeed. This whole ns needs a refactor. Too many opportunities to get confused.